.coveragerc_py38

```diff
@@ -5,13 +5,13 @@ timid = True
 [report]
 exclude_lines =
     pragma: no cover
-    pragma: py2 no cover
-    if six.PY3
-    elif six.PY3
+    pragma: py3 no cover
+    if six.PY2
+    elif six.PY2
 
 partial_branches =
     pragma: no cover
-    pragma: py2 no cover
+    pragma: py3 no cover
     if six.PY3
```

Comment:
Should it be perhaps be?
```
if six.PY2
elif six.PY2
```
If not, this is a bit confusing - could we add comment to explain what it means...

Reply:
@nskool Not really sure where this pragma comes from. I just kept the configuration the same as other py3 versions